cmd/apps/traefik2.go

```diff
@@ -0,0 +1,164 @@
+		if command.Flags().Changed("kubeconfig") {
+			kubeConfigPath, _ = command.Flags().GetString("kubeconfig")
+		}
+		fmt.Printf("Using kubeconfig: %s\n", kubeConfigPath)
+		//arch := getNodeArchitecture()
+		//fmt.Printf("Node architecture: %q\n", arch)
+		updateRepo, _ := traefik2.Flags().GetBool("update-repo")
+		namespace, _ := traefik2.Flags().GetString("namespace")
+		userPath, err := config.InitUserDir()
+		if err != nil {
+			return err
+		}
+
+		_, clientOS := env.GetClientArch()
+		clientArch, clientOS := env.GetClientArch()
+		fmt.Printf("Client: %q\n", clientOS)
+
+		_, err = helm.TryDownloadHelm(userPath, clientArch, clientOS, false)
+		if err != nil {
+			return err
+		}
+
+		err = addHelmRepo("traefik", "https://containous.github.io/traefik-helm-chart", false)
+		if err != nil {
+			return fmt.Errorf("Unable to add repo %s", err)
+		}
+
+		if updateRepo {
+			err = updateHelmRepos(false)
+			if err != nil {
+				return err
+			}
+		}
+
+		chartPath := path.Join(os.TempDir(), "charts")
+		fmt.Printf(chartPath)
+		err = fetchChart(chartPath, "traefik/traefik", "", false)
+		if err != nil {
+			return fmt.Errorf("Unable fetch char %s", err)
+		}
+
+		overrides := map[string]string{}
+		overrides["service.type"] = "NodePort"
+
+		dashboard, _ := command.Flags().GetBool("dashboard")
+		if dashboard {
+			overrides["dashboard.ingressRoute"] = "true"
+		}
+
+		customFlags, err := command.Flags().GetStringArray("set")
+		if err != nil {
+			return fmt.Errorf("error with --set usage: %s", err)
+		}
+
+		if err := mergeFlags(overrides, customFlags); err != nil {
+			return err
+		}
+
+		outputPath := path.Join(chartPath, "traefik/traefik/")
+		err = templateChart(chartPath,
+			"traefik",
+			namespace,
+			outputPath,
+			"values.yaml",
+			overrides)
+
+		if err != nil {
+			return err
+		}
+
+		err = kubectl("apply", "-R", "-f", outputPath, "-n", namespace)
+		if err != nil {
+			return err
+		}
+		fmt.Println(traefikInstallMsg)
+		return nil
+	}
+
+	return traefik2
+}
+
+func PrintTraefikASCIIArt() {
+	arkadeLogo := aec.BlueF.Apply(traefikstarted)
+	fmt.Print(arkadeLogo)
+}
+
+const traefikstarted = `
+
+_______              __ _ _           ___  
+|__   __|            / _(_) |         |__ \ 
+   | |_ __ __ _  ___| |_ _| | __ __   __ ) |
+   | | '__/ _´ |/ _ \  _| | |/ / \ \ / // / 
+   | | | | (_| |  __/ | | |   <   \ V // /_ 
+   |_|_|  \__,_|\___|_| |_|_|\_\   \_/|____|
+
+`
+const traefikInstallMsg = `=======================================================================
+=                  traefik v2 has been installed.                        =
+=======================================================================
+` +
+	"\n" + pkg.ThanksForUsing +
+	`
+NOTES:
+
+1. Traefik has been started. You can find out the port numbers being used by traefik by running:
+
+          $ kubectl describe svc traefik --namespace kube-system
+
+2. To Access to Dashboard you can run kubectl proxy or expose the service, for example:
```

Comment:
@braybaut Maybe we can write put the and in the next line, so the user can copy and paste the command in the terminal?